### PR TITLE
Adding a reproducer

### PR DESCRIPTION
**Pull request by pmlopes**
Signed-off-by: Paulo Lopes <pmlopes@gmail.com>

Motivation:

Trying to verify issue with pg

### CODE REVIEW

File: src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java [hunk 1]
```diff
@@ -33,20 +33,6 @@
...
 public interface JDBCColumnDescriptorProvider {
 
-  /**
-   * Create provider by parameter statement
-   *
-   * @param statement the prepared statement
-   * @return a new {@code JDBCTypeProvider} instance
-   * @see java.sql.PreparedStatement
-   */
-  static JDBCColumnDescriptorProvider fromParameter(PreparedStatement statement) {
```
Comment (ruslansennov) [inline src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java:43]: @pmlopes  You removed it completely, thanks! :)

Comment (pmlopes) [reply src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java:43]: @ruslansennov sorry if you miss it. I was under the impression we can achieve the same with:

```
ParameterMetaData md = new CachedParameterMetaData(statement);
JDBCColumnDescriptorProvider provider = JDBCColumnDescriptorProvider.fromParameterMetaData(md);
```

I should add it back as this was an SPI not an impl

Comment (pmlopes) [reply src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java:43]: It's reverted!

Comment (ruslansennov) [reply src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java:43]: @pmlopes  no! My personal opinion: it was the right decision :) But this method should be used very carefully.

**Pull request merged**